### PR TITLE
Editor: add payments button step to Editor Basics Tour

### PR DESCRIPTION
**Pull request by lsinger**
This PR adds a step to the "Editor Basics Tour" that tells people about Simple Payments. 

Screenshot of the new step:

screen shot 2017-08-16 at 1 53 15 pm

To test:
- go to http://calypso.localhost:3000/post/SITE_URL?tour=editorBasicsTour
- see whether the new step is rendered correctly and check its copy
- make sure the tour as a whole makes sense

### CODE REVIEW

**Comment by michelleweber** [comment]
We usually use "Business plan" as a name and not just a description, so I'd change this to "If you have the Business plan..."

(Also, while tweaking: it'd be great to change "such as" to "like" -- it's a bit more conversational.)

**Comment by rralian** [comment]
This looks good to me. Let's merge this tomorrow when we publish the announcement post.

File: client/layout/guided-tours/tours/editor-basics-tour.js [hunk 1]
```diff
@@ -83,6 +83,30 @@ export const EditorBasicsTour = makeTour(
...
+		<Step
+			name="add-other"
+			arrow="top-left"
+			target=".editor-html-toolbar__button-insert-media, .mce-wpcom-insert-menu button"
+			placement="below"
+			style={ { marginLeft: '22px', zIndex: 'auto' } }
+		>
+			<p>
+				{
+					translate( 'The {{icon/}} lets you add other things, like a contact form. ' +
```
Comment (spen) [inline client/layout/guided-tours/tours/editor-basics-tour.js]: Just a very minor, non-blocking nitpick here:
The indentation seems a little off to me, how do you feel about:

```jsx
translate( 
  'The {{icon/}} lets you add other things, like a contact form. ' +
  'If your site is on the Premium or Business plan, you can even add {{strong}}payment buttons{{/strong}}!', 
  {
    components: {
      ...
    }
  }
```
Rational: 
- keeps the text grouped together (without unrelated syntax sharing the same line. It's boundaries are easier to see at a glance.
- the { of the attributes object doesn't get lost at the end of the 2nd line of text and instead is more obvious.

Comment (lsinger) [inline client/layout/guided-tours/tours/editor-basics-tour.js]: perfect! thanks!

**Comment by rralian** [comment]
I rebased this and am approving. Will merge after our announcement post launches.

**Pull request merged**